Gemfile

```diff
@@ -27,9 +27,11 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.2', require: false
 gem 'bootstrap', '~> 4.4.1'
+gem 'haml-rails', '>= 0.3.4', :group => :development
```

Comment:
@ValeriaMejiaCortes when adding a new gem, you may want to consider using the latest version compatible with your stack of technologies. Taking a look at the library [repo](https://github.com/haml/haml-rails) it seems that the current version is 2.0, which I think should play well with the ruby/rails versions you're using. Please update it accordingly as 0.3.4 is kind of a pretty old version.